src/main/java/no/nav/pto/veilarbportefolje/kafka/KafkaCommonConsumerService.java

```diff
@@ -0,0 +1,43 @@
+package no.nav.pto.veilarbportefolje.kafka;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.pto.veilarbportefolje.service.UnleashService;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+
+import java.util.concurrent.TimeUnit;
+
+import static no.nav.common.client.utils.CacheUtils.tryCacheFirst;
+import static no.nav.pto.veilarbportefolje.config.FeatureToggle.NY_KAFKA_COMMON_LIB;
+
+@Slf4j
+public abstract class KafkaCommonConsumerService<T> {
+    private final Cache<String, Boolean> unleashServiceCache = Caffeine.newBuilder()
+            .expireAfterWrite(30, TimeUnit.SECONDS)
+            .maximumSize(10)
+            .build();
+
+    protected boolean isNyKafkaLibraryEnabled() {
```

Comment:
Tror ikke det er nødvendig å ha cache for Unleash. `Unleash`-klienten har en cache innebygd.

Reply:
Kunne ikke finne en cache i Unleash klienten :(

Reply:
Fixed, thanks for help!